# POGO-DB: Database of Pairwise-comparisons Of Genomes and universal Orthologous genes
POGO-DB (http://pogo.ece.drexel.edu) aims to provide biologists and bioinformaticians with an user-friendly web interface as well as API to compare microbial genomes currently sequenced and stored in the NCBI complete genome database. The project was inspired by the variant evolution rate of genes which made it challenging to reconstruct the relationships between microbes based on one particular marker gene. 

# Quantifications Provided
POGO-DB allows users to compare microbial genomes using pre-computed metrics that were derived from extensive computationally intensive comparisons of >2000 microbes, and help decide optimal marker genes for any user-selected microbial group that would best reconstruct the microbial relationship groundtruth:
(i) average protein sequence identity across all orthologs shared by two genomes;
(ii) genomic fluidity (a measure of gene content dissimilarity);
(iii) number of ‘orthologs’ shared between two genomes;
(iv) pairwise identity of the 16S ribosomal RNA genes;
(v) pairwise identity of an additional 73 marker genes present in >90% prokaryotes.

# Current Version
The current release of POGO-DB is based on genomes of 2,013 bacteria strains from the NCBI database (as of available in July, 2012). Update is in progress to incorporate 910 more bacteria strains (as of available in February, 2015).

# Credits
The manuscript has been published on Nucleic Acids Research (http://nar.oxfordjournals.org/content/early/2013/11/05/nar.gkt1094)by:
Yemin Lan
J. Calvin Morrison
Ruth Hershberg
Gail L. Rosen

